docs/howto_guides/reproject_vbi_moasic.md

```diff
@@ -0,0 +1,148 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Stitching a VBI Moasic with `reproject`
+
+The [reproject](https://reproject.readthedocs.io/) package is an Astropy-affiliated package for regridding data.
+A number of different algorithms are implemented in the package, with different trade-offs for speed and accuracy.
+Reprojecting a single spatial image such as an AIA image is well supported and demonstrated in the [sunpy gallery](https://docs.sunpy.org/en/latest/generated/gallery/index.html#combining-co-aligning-and-reprojecting-images).
+When working with DKIST data, there are normally many spatial frames inside a dataset, and reprojecting them all can be very computationally expensive.
+
+We are going to use the example of using reproject's {obj}`reproject.mosaicking.reproject_and_coadd` function to stitch a mosaic of VBI frames, in a way which could be extended to be flux conserving.
+
+
+## Obtaining some data
+
+```{code-cell} ipython3
+import numpy as np
+import matplotlib.pyplot as plt
+import astropy.units as u
+
+from sunpy.net import Fido, attrs as a
+
+import dkist
+import dkist.net
+```
+
+```{code-cell} ipython3
+res = Fido.search(a.dkist.Dataset("AJQWW"), )
+res
+```
+
+Let's download a VBI dataset, which has good seeing:
+
+```{code-cell} ipython3
+asdf_file = Fido.fetch(res, path="~/dkist_data/{dataset_id}")
+asdf_file
+```
+
+This gives us a `TiledDataset` object, which as we have already seen is like an array of `Dataset` objects.
+
+```{code-cell} ipython3
+ds = dkist.load_dataset(asdf_file)
+ds
+```
+
+This mosaic is 3 x 3:
+
+```{code-cell} ipython3
+ds.shape
+```
+
+To start off, let's just download the first frame of each tile.
+
+```{code-cell} ipython3
+first_tiles = [tile[0] for tile in ds.flat]
+```
+
+```{code-cell} ipython3
+:tags: [skip-execution]
+
+for i, tile in enumerate(first_tiles):
+    tile.files.download(wait=i == len(first_tiles) - 1)
```

Comment:
Maybe add a comment here explaining what the wait argument is doing in this case, unless this is demonstrated elsewhere in the docs. Took me a minute to parse this.